### ISSUE TITLE
Can`t login with PokeBuddy

### ISSUE THREAD

**Issue by karimavine** [finndev/PokeBuddy#1085]
i type in the id/pass when i press login nothing happens 
also antivirus is off

**Comment by darrensee** [comment]
same here

**Comment by kobayashiishin** [comment]
same

**Comment by bankrecon44** [comment]
PTC already bought it?

**Comment by Zhamint** [comment]
same here

**Comment by xxbahaxx** [comment]
Same! I think the sever might be down! Anyone elselse is using pokebuddy right now?

**Comment by 273849906** [comment]
same

**Comment by vaizard12** [comment]
pokebuddy site is down. Maybe suspended?

**Comment by darrensee** [comment]
say bye to pokebuddy LOL

**Comment by 273849906** [comment]
LOL, CANNOT LOG IN:)

**Comment by nickchangatc1** [comment]
Looks like login server is down good night

**Comment by Kar1m0v** [comment]
looks like site is down, cause i have "error 522"

**Comment by yeapeh** [comment]
<img width="787" alt="error" src="https://cloud.githubusercontent.com/assets/20989887/17648080/dee5859e-623b-11e6-95e8-b901988aba5c.png">

same here looks like server down,

**Comment by likachuntroy** [comment]
Same here

**Comment by 273849906** [comment]
WHATS WRONG

**Comment by Rab89** [comment]
guys that site has no affiliation with pokebuddy or finndev, its most likely down because the scammer has pulled it due to the recent legal actions some bots devs have had, 

thats what the dude who stole the bot and tried to claim as his own gets :laughing:

**Comment by grimgaze** [comment]
I was using it , and its working perfectly fine before i accidentally closed it and updated, now need an account and cant create an account. And btw. Thank u. Found the answer i was looking for. Yep it might be the website is down . Any ideas when will be up and running again?

**Comment by Mekanizm** [comment]
Ok. It would be nice if someone who really knows something would come and say what is really going on and if the bot is still usable and what kind of account works on it.

**Comment by grimgaze** [comment]
Rab89, yup. I saw a message "under attack" i already aware of this since i was running a website before. Their hacking into its system. It might take a couple of hours to block a nerd out.

**Comment by Rab89** [comment]
That site wont be back up

**Comment by grimgaze** [comment]
what!? So the bot wont work anymore -.- cant create account for it -.-

**Comment by N1ghtmaree** [comment]
Guys dont worry, just wait for it everything is working within 24hrs

**Comment by N1ghtmaree** [comment]
working again*

**Comment by Rab89** [comment]
Nope pokebuddy is dust along with majority of the bots, only one i know of still accessible is Necrobot, id post the link but the dl link has been removed from an earlier post on here

**Comment by N1ghtmaree** [comment]
Okay

**Comment by Rab89** [comment]
also the link on necrobot devs github has been pulled fyi

**Comment by grimgaze** [comment]
So ur saying , pokebuddy is a gonner now?

**Comment by N1ghtmaree** [comment]
Im almost sure it will come back online

**Comment by Rab89** [comment]
yep i noticed the readme file had been updated with a legal disclaimer and then 6 hours later the bots inaccessible :(

**Comment by 273849906** [comment]
so it wont working forever?

**Comment by N1ghtmaree** [comment]
The site is just getting a ddos...
Its nothing to worry about, the bots runs via the site.

**Comment by N1ghtmaree** [comment]
Or under maintentance

**Comment by grimgaze** [comment]
Well, one thing for sure.' Wait and see

**Comment by N1ghtmaree** [comment]
Yep...

**Comment by 273849906** [comment]
well

**Comment by Rab89** [comment]
thats very true i hope it works for you guys but it isnt looking good

**Comment by N1ghtmaree** [comment]
Rab98 
Do you know another free pokemon bot?

**Comment by 273849906** [comment]
when i log out it can not log in anymore, but before log out it works well

**Comment by N1ghtmaree** [comment]
Everyone cant login atm

**Comment by grimgaze** [comment]
It does, before it updated itself.

**Comment by Rab89** [comment]
i do :+1: 

ill post a link on my profile

**Comment by 273849906** [comment]
if i have not log out can it work?

**Comment by N1ghtmaree** [comment]
Okay rab

**Comment by N1ghtmaree** [comment]
No 27849906 the bots are running on the site and the site is down

**Comment by 273849906** [comment]
thksN1ghtmaree

**Comment by grimgaze** [comment]
Rab. Is it just same as pokebuddy? And afraid of niantic might trace it and completely ban my account for it. xD

**Comment by N1ghtmaree** [comment]
Just use vpn grimgaze ;p

**Comment by grimgaze** [comment]
I still doubt about it. But who knows.

**Comment by grimgaze** [comment]
Where is rab. Waiting for his post :p

**Comment by likachuntroy** [comment]
Just use the vpn and login with pokebuddy? It will be work?

**Comment by N1ghtmaree** [comment]
No likeachuntroy

**Comment by N1ghtmaree** [comment]
grimgaze me2 ;p

**Comment by Rab89** [comment]
ive been using it for the last 3 days without any hiccups,
good news i found the link the dude posted the other day. the UI isnt as good as Pokebuddy but it works and has a few things like auto incubating that pokebuddy doesnt have. 
once you download it open the Necroconfig.exe and put in your sign in details.
then run necrobot.exe
better be quick this link will be pulled shortly
https://jumpshare.com/v/HaOuumzsYDkq1S4GtmVQ

**Comment by N1ghtmaree** [comment]
Rab my chrome blocks it?

**Comment by N1ghtmaree** [comment]
virus?

**Comment by N1ghtmaree** [comment]
Nvm i opened it with explorer and now its fine? xd

**Comment by Rab89** [comment]
yeah chrome does block it but i can guarantee you its only a false negative.

PROOF
http://prntscr.com/c5ildl

**Comment by GabrielAtanasov** [comment]
![image](https://cloud.githubusercontent.com/assets/14962417/17648369/c9cd2dea-621a-11e6-884a-c7defd2313df.png)

**Comment by Rab89** [comment]
Enjoy guys its the only free one i know still works and not to mention the dev pulled the bot from his github page

**Comment by N1ghtmaree** [comment]
Do you have to use human walk or not?

**Comment by GabrielAtanasov** [comment]
Rab89
I did not understand - PokeBuddy will be there online again or permanently stops working?

**Comment by N1ghtmaree** [comment]
Gabriel we dont know...

**Comment by Rab89** [comment]
![image](https://cloud.githubusercontent.com/assets/20825714/17648386/971fc228-620a-11e6-926d-8ab970c9edc1.png)

**Comment by N1ghtmaree** [comment]
Thx rab

**Comment by Rab89** [comment]
no problem buddy, glad to help out

**Comment by karimavine** [comment]
@Rab89 
https://s4.postimg.io/9u698f2ul/Untitled.png
do i need a certain programm to be installed for this exe?

**Comment by N1ghtmaree** [comment]
It works fine, ty rab :)

**Comment by grimgaze** [comment]
gonna set up my VPN then. hahaha

**Comment by N1ghtmaree** [comment]
Use soft ether vpn client manager its one of the best free vpns

**Comment by Rab89** [comment]
@karimavine no it should be fine just ensure your .NET framework is uptodate, a quick google search will fix that :)
@grimgaze i dont even use a VPN lol

**Comment by N1ghtmaree** [comment]
omg shit necrobot transfered all my pokemon for some reason

**Comment by Rab89** [comment]
:sob:  you're kidding

**Comment by N1ghtmaree** [comment]
nope
:(

**Comment by karimavine** [comment]
@Rab89 doesn`t it work on the same net frame as pokebuddy?

**Comment by N1ghtmaree** [comment]
OMG yes i closed at the good moment i still have my 9 highest pokemons

**Comment by Rab89** [comment]
it should work screenshot what happens when you click it.
and omg nightmaree favourite them sorry to hear that bud

**Comment by N1ghtmaree** [comment]
xd

**Comment by Rab89** [comment]
ohhhh did i mention the bot unsoftbans you aswell :D

**Comment by karimavine** [comment]
@Rab89 it just says "choose a program you want to use to open this file "

**Comment by Rab89** [comment]
its a .exe file any standard computer can run that

**Comment by N1ghtmaree** [comment]
how much iv% should i auto favorite i dont know what it means so ;p

**Comment by Rab89** [comment]
i put mine to 80%

**Comment by N1ghtmaree** [comment]
ty

**Comment by Rab89** [comment]
@karimavine your screenshot showed the necrobot.exe as an xml file 
it should be 
![image](https://cloud.githubusercontent.com/assets/20825714/17648422/bd0978f6-620c-11e6-816f-097afcc8e9b7.png)

**Comment by Rayquazaaa** [comment]
Mmmm. We shouldn't even need accounts to do this.

**Comment by Rab89** [comment]
also make sure you edit the transfer tab in the necroconfig.exe with the pokemon you dont want to transfer.

**Comment by N1ghtmaree** [comment]
rab can you upload your config file and send link pls? xd

**Comment by Rab89** [comment]
give me a minute then :D

**Comment by massnigg** [comment]
The pokebuddy isn't working for me

**Comment by N1ghtmaree** [comment]
k rab :D

**Comment by Rab89** [comment]
here you go :D 
http://www.mediafire.com/download/4zwukmxxtqjfm17

**Comment by N1ghtmaree** [comment]
Ty man :)

**Comment by Rab89** [comment]
no problemo

**Comment by sierensthibault** [comment]
i downloaded necro bot but it says it can not open it

**Comment by Rab89** [comment]
screenshot it @sierens

**Comment by massnigg** [comment]
The download doesn't work
I can't open file

**Comment by Rayquazaaa** [comment]
Uh. Necrobot shut down m8.

**Comment by massnigg** [comment]
I guess bots are all gone from pokemon go

**Comment by N1ghtmaree** [comment]
Im using necrobot atm and its working fine...

**Comment by Rab89** [comment]
yeah the official devs github has but it still works if you already have the necrobot

**Comment by Rab89** [comment]
i posted a link above 41 mins ago be quick though its gonna be pulled soon

**Comment by Rab89** [comment]
nghtmare did you see what i posted in the config file lol

**Comment by grimgaze** [comment]
N1ghtmare can u post a screen shot that its working? Can seem to work on mine

**Comment by Rayquazaaa** [comment]
Well I just issued a ban appeal. Got perma banned. Stop using bots NOW.

**Comment by sierensthibault** [comment]
it just says "there is  problem and the program doesn't work well. The program will be closed"

**Comment by sierensthibault** [comment]
![image](https://cloud.githubusercontent.com/assets/20890609/17648513/5729e3c0-6218-11e6-977f-7a0bb2e630bc.png)

**Comment by N1ghtmaree** [comment]
Here grimgaze 
![image](https://cloud.githubusercontent.com/assets/21019669/17648518/662083b6-6218-11e6-9d26-5e331fb8d7eb.png)

**Comment by massnigg** [comment]
All bots have been removed. I guess we have to play legitly now

**Comment by N1ghtmaree** [comment]
Thats my second screen left btw dont look at it ;p

**Comment by Rayquazaaa** [comment]
Guys. Just stop using bots. I got perma banned.

**Comment by Rab89** [comment]
bans are imminent but until i get banned i wont stop.
ive botted since pokebuddy was released no vpn nothing and still bot.

and im working completely fine sierens
![image](https://cloud.githubusercontent.com/assets/20825714/17648524/40ead810-6210-11e6-9d68-724e617ba740.png)

**Comment by massnigg** [comment]
How do you know it's a perma not a soft???

**Comment by Rayquazaaa** [comment]
content://media/external/file/318

**Comment by Rayquazaaa** [comment]
Well that was odd

**Comment by Rab89** [comment]
rayquazaa doesnt know if hes perma banned lol

**Comment by massnigg** [comment]
I don't know either but looking at youtube you can't even log on and it will tell you if you are perma banned

**Comment by Rayquazaaa** [comment]
Sure. I totally dont have the fail to get game info from server error.

**Comment by sierensthibault** [comment]
every time i open it it says it does't work anymore

**Comment by N1ghtmaree** [comment]
I dont even think perm bans are in the game ;p

**Comment by Rayquazaaa** [comment]
![screenshot_2016-08-14-06-15-43](https://cloud.githubusercontent.com/assets/15619708/17648528/c871f1f8-61e6-11e6-806b-2b8b9c33a79b.png)
See I'm banned.

**Comment by Rab89** [comment]
perm bans have not been made official all hearsay until i get an email from niantic themselves stating a perma ban i dont and wont believe anyone.

**Comment by massnigg** [comment]
just means that you aren't connected to their servers

**Comment by Rab89** [comment]
rayquazaa close some god damn apps your phones about to blow up

**Comment by Rayquazaaa** [comment]
![screenshot_2016-08-14-06-18-25](https://cloud.githubusercontent.com/assets/15619708/17648531/eef2eb98-61e6-11e6-8854-7729f50e7588.png)
Here, for you nonbelievers.

**Comment by N1ghtmaree** [comment]
Wow, that sucks

**Comment by Rab89** [comment]
-NianticCops

**Comment by massnigg** [comment]
SUCKS TO BE YOU

**Comment by massnigg** [comment]
SUCKS TO BE YOU

**Comment by N1ghtmaree** [comment]
they mean operators with OPS ;p

**Comment by Rayquazaaa** [comment]
I just realized rab doesn't know what notifications are.

**Comment by grimgaze** [comment]
do i still need to turn off my anti virus for it?
mine isnt working -_-

**Comment by N1ghtmaree** [comment]
just bot another account and stop at level 25 or something you wont get banned if you do it good

**Comment by N1ghtmaree** [comment]
grimgaze download with Explorer not with chrome than it works

**Comment by Duero421** [comment]
@Rab89  the config file you uploaded... nothing pops out when i open it can you help me a bit?

**Comment by Rayquazaaa** [comment]
Just give up on botting. Unless you want to be perma banned.

**Comment by Rab89** [comment]
im not bothered rayquazaaaaaa

**Comment by grimgaze** [comment]
![1](https://cloud.githubusercontent.com/assets/21019537/17648546/ced8993a-61d6-11e6-89a9-1531ea07487a.jpg)

**Comment by Rab89** [comment]
@duero i posted a jumpshared link earlier in this thread

**Comment by N1ghtmaree** [comment]
Rayquazaa i botted 2 accounts to level 20 and only 1 fucking soft-ban received you are doing somethng wrong man...

**Comment by Rab89** [comment]
im level 30 and botted since pokebuddy was released and not once got an ip ban

**Comment by grimgaze** [comment]
rab mine isnt working dnt know whats wrong with it

**Comment by Rayquazaaa** [comment]
Please m8 I have like 6 dragonites. Thats obviously a hack/bot account.

**Comment by N1ghtmaree** [comment]
But you are alot safer of ipbans with a vpn than they only ban that vpn...

**Comment by grimgaze** [comment]
well its ur fault. i abused it

**Comment by Duero421** [comment]
@Rab89  thanks man you're the best

**Comment by Rayquazaaa** [comment]
God dammit just shut the fuck up. I just told you that stopping would be a good idea.

**Comment by Rab89** [comment]
i have 2 charizards 3 arcanines 4 vaporeons 3 dragonites all over 2000 cp

**Comment by Rayquazaaa** [comment]
Now you have to defend botting like I questioned your religion or sanity.

**Comment by Rab89** [comment]
rayquazaaa dont be jel because we can still bot now piss off lol and take your scaremongering elsewhere

**Comment by grimgaze** [comment]
![1](https://cloud.githubusercontent.com/assets/21019537/17648564/563acdbc-61d7-11e6-82b5-8b4e1373f844.jpg)

rab can u help me.

**Comment by N1ghtmaree** [comment]
Rayquazaaa you are probably about 11 years old or arent you?

**Comment by Rayquazaaa** [comment]
Have fun when your perma banned ;)))))))))

**Comment by Rayquazaaa** [comment]
You have been warned.

**Comment by Duero421** [comment]
LOL hater

**Comment by N1ghtmaree** [comment]
Rayquazaaa have fun creating a new account :)

**Comment by Rab89** [comment]
grim i apologize but i dont have a clue on a workaround for that i know pokebuddy done the same thing.
and ray oh we will its called make another fake account lol

**Comment by Rayquazaaa** [comment]
Ok cool

**Comment by N1ghtmaree** [comment]
Lol i can just do 500km/h and still no softban after 10mins lol

**Comment by Rab89** [comment]
thats me now at 135 of 151 pokemon for my pokedex :raised_hands:

**Comment by Rab89** [comment]
yep nightmaree but i just put it down to 40 like i did with pokebuddy

**Comment by N1ghtmaree** [comment]
What are you actually gonna do with that account? ;p

**Comment by Rab89** [comment]
complete my pokedex and find out a way to get mew lol i reckon its gonna be something really silly like lvl 40 all medals gold all pokemon complete and it spawns for you lol

just a theory though lol

**Comment by N1ghtmaree** [comment]
;pppp

**Comment by Duero421** [comment]
@Rab89 Hey rab can i borrow your config file? :)

**Comment by N1ghtmaree** [comment]
Rab89 is there a way the bot claims gyms for me?

**Comment by sierensthibault** [comment]
does somebody know why i cant open necrobot ?

**Comment by owenznaga** [comment]
POKEMON GO IS UPDATED AND BLOCK ALL BOT

**Comment by N1ghtmaree** [comment]
@owenznaga no stfu

**Comment by Rab89** [comment]
nightmare nope im afraid.
duero sure 
at owen no its not im botting just now lol
at sierens its a strange one the necrobot.exe should be a application and not an xml file

**Comment by Rab89** [comment]
@duero421 here you go
http://www.mediafire.com/download/4zwukmxxtqjfm17/NecroConfig.exe

**Comment by N1ghtmaree** [comment]
and rab is there a way to see your screen of the app? ;p

**Comment by Duero421** [comment]
@Rab89 
thanks dude

**Comment by Rayquazaaa** [comment]
https://youtu.be/4g9zsRGoAUo mmm fresh smell of you guys being wrong.

**Comment by Rab89** [comment]
Sure N1ghtmaree :P 

![image](https://cloud.githubusercontent.com/assets/20825714/17648603/aaf68bb6-6213-11e6-9df3-744685e98370.png)

**Comment by N1ghtmaree** [comment]
i mean like you see the pokemon go app on your mobile;ppppp

**Comment by Rab89** [comment]
ohhh i just got  a new phone and dont have a screengrab app yet XD

**Comment by Rab89** [comment]
give me a few mins ill get one lol

**Comment by owenznaga** [comment]
LOL PERMA BANNED

**Comment by N1ghtmaree** [comment]
from what?

**Comment by Rab89** [comment]
shut up owen hahahahahahaha

![image](https://cloud.githubusercontent.com/assets/20825714/17648639/aa1f7e2c-6214-11e6-89e1-322cedf61762.png)

**Comment by Rab89** [comment]
ohhhh loook at that rayquazaaaa and owen a working account lol

**Comment by N1ghtmaree** [comment]
Nice

**Comment by N1ghtmaree** [comment]
Im level 22 yesterday i started with botting from level 1 ;p

**Comment by N1ghtmaree** [comment]
and only did about 5 hrs in total

**Comment by owenznaga** [comment]
Your necro bot cannot opened --_--

**Comment by Rab89** [comment]
nice you level up really fast until lvl 20 , lvl 30 is even worse takes ages to level up

**Comment by Rab89** [comment]
clearly does from my previous screenshot lol you still here owen lol

**Comment by owenznaga** [comment]
yep
haha

**Comment by massnigg** [comment]
My botted account still works.

**Comment by Duero421** [comment]
![untitled](https://cloud.githubusercontent.com/assets/21020455/17648655/09a9116a-6250-11e6-95bf-8a40afbb0b44.png)

it's stuck there.. is there a problem or is it normal? i am new to necrobot :( last bot i used was pokebuddy

**Comment by Rab89** [comment]
if your so against botting owen  why are you trying to download it lol
and duero close and open again

**Comment by rs232d** [comment]
Rab, you realize you have posted a rar file with your google account and your password ?. look at the configuration files

**Comment by Duero421** [comment]
Thanks again Rab -P

**Comment by N1ghtmaree** [comment]
*rab changing passwords and shit ;p*

**Comment by Rab89** [comment]
thats not mine @rs232d that was the guy who posted the link before lol

**Comment by N1ghtmaree** [comment]
wow i just caught pickachu, no i see its fucking 50cp...

**Comment by Duero421** [comment]
Any chance of pokebuddy working again?

**Comment by N1ghtmaree** [comment]
Maybe

**Comment by Rab89** [comment]
it toook you thiss long nightmare to notice lol
and nope its not looking good duero

**Comment by Duero421** [comment]
I see :( i prefer pokebuddy since it's easy to understand

**Comment by Rab89** [comment]
yeah i agree duero necrobots UI isnt as appealing tbh as pokebuddys but its a working bot

**Comment by Duero421** [comment]
BTW is your config file safe to use? i mean looking at it's configuration it seems like it's walking speed is too fast

**Comment by Rab89** [comment]
necrobots walking speed works at 500km/h lol
40 was pokebuddys default

**Comment by Duero421** [comment]
OHHH o.o thanks again

**Comment by N1ghtmaree** [comment]
im running it on 120km/h for 30mins now now softbans still :)

**Comment by Rab89** [comment]
https://jumpshare.com/v/HaOuumzsYDkq1S4GtmVQ
link to necrobot

NOTE- the login details arent mine but was the kind uploaders, who posted a few days back,
thanks and enjoy

**Comment by Rab89** [comment]
i think my works done now someone else can help with your questions (**N1ghtmaree**)

**Comment by N1ghtmaree** [comment]
im softbanned xd run into 900km/h ;p

**Comment by Duero421** [comment]
LOL play it safe hahahaha

**Comment by sierensthibault** [comment]
what is this ?
![image](https://cloud.githubusercontent.com/assets/20890609/17648737/ce9c70f0-6220-11e6-9ede-bfcf43e87050.png)

**Comment by Duero421** [comment]
Wrong acc probably

**Comment by Rab89** [comment]
you need to open the necroconfig.exe and edit the sign in lol

**Comment by Rab89** [comment]
tell me what it says sierens :laughing:

**Comment by N1ghtmaree** [comment]
Maybe you should enter your information before starting 0_0

**Comment by N1ghtmaree** [comment]
Rab where do you come from?

**Comment by Rab89** [comment]
im from the UK and yep exactly that

**Comment by LeoAchilles** [comment]
the PokeBuddy website works again, and is able to sign up account

**Comment by N1ghtmaree** [comment]
yaaaay

**Comment by Rab89** [comment]
lol niiice

**Comment by Duero421** [comment]
Niiiiceeeeeee!!!!!!!!!!!!!!!!!

**Comment by N1ghtmaree** [comment]
It was an update i guess my pokebuddy is much faster and no crashes now

**Comment by Duero421** [comment]
I'm keeping this necrobot for backup hehehe

**Comment by N1ghtmaree** [comment]
y me2

**Comment by N1ghtmaree** [comment]
Pokebuddy has much errors, ill use necrobot if there is an error again ;p

**Comment by UncleAss** [comment]
thanks Rob,  i love Necrobot :)  i have a question for you.. i configured api telegram but it doesn works. Necrobot sais

**Comment by UncleAss** [comment]
.. says with a red log that there is an error

**Comment by N1ghtmaree** [comment]
Whats api telegram?

**Comment by dzdam** [comment]
is there some improvement in configuration setup? like delay between actions?

**Comment by N1ghtmaree** [comment]
yes

**Comment by sierensthibault** [comment]
it keeps saying this ??
![image](https://cloud.githubusercontent.com/assets/20890609/17648899/cfffc474-6225-11e6-94e6-ef5e2737bb22.png)

**Comment by N1ghtmaree** [comment]
close and open again

**Comment by N1ghtmaree** [comment]
did that fixed the problem?

**Comment by sierensthibault** [comment]
nope

**Comment by N1ghtmaree** [comment]
but your maximum walk distance to 999999

**Comment by N1ghtmaree** [comment]
put*

**Comment by N1ghtmaree** [comment]
Did that fix it?

**Comment by sierensthibault** [comment]
nope

**Comment by N1ghtmaree** [comment]
If not try to close necrobot login with your phone to pokemongo, if you cant you are probably banned

**Comment by sierensthibault** [comment]
![image](https://cloud.githubusercontent.com/assets/20890609/17648922/755831b8-6226-11e6-823e-3fa69acba612.png)

**Comment by N1ghtmaree** [comment]
Put delay between actions 5000 both of them

**Comment by N1ghtmaree** [comment]
set your current coords to default and than open necrobot again

**Comment by N1ghtmaree** [comment]
fixed?

**Comment by sierensthibault** [comment]
still nothing :(

**Comment by sierensthibault** [comment]
and not banned

**Comment by N1ghtmaree** [comment]
mmmm

**Comment by N1ghtmaree** [comment]
try to redownload the files of necrobot

**Comment by N1ghtmaree** [comment]
I have necrobot active for about 3 hours now and had 0 errors

**Comment by UncleAss** [comment]
no one can help me with telegram bot on necrobot?  D:

**Comment by N1ghtmaree** [comment]
Uncleass i dont even no what it is, you dont need to use it

**Comment by sierensthibault** [comment]
can i see your settings

**Comment by N1ghtmaree** [comment]
![image](https://cloud.githubusercontent.com/assets/21019669/17648975/13395f00-6228-11e6-8dff-23299d4565be.png)
Here

**Comment by UncleAss** [comment]
yes i need it.  i had created a telegram bot last week, that told me stats profile etc on telegram of the working necrobot. 
with this necrobot installed today i can t do that

**Comment by sierensthibault** [comment]
still not working thanks for the help im giving up

**Comment by Duero421** [comment]
Try pokebuddy.. it's working now :)

**Comment by N1ghtmaree** [comment]
fixed?

**Comment by Duero421** [comment]
![untitled](https://cloud.githubusercontent.com/assets/21020455/17649014/45b6ab44-625b-11e6-87af-807c63eeebb8.png)

i guess it's fixed? :D

**Comment by sierensthibault** [comment]
my pokebuddy doesnt work i do not walk and it says updating pokestops

**Comment by tkkcc667788** [comment]
they banned the IP, use a VPN then launch pokebuddy

**Comment by N1ghtmaree** [comment]
im using vpn y i think thats the problem of him

**Comment by tkkcc667788** [comment]
whenever i get the error i change my IP using my VPN

**Comment by sierensthibault** [comment]
someone a good vpn ?

**Comment by N1ghtmaree** [comment]
But guys necrobot is much faster, pokebuddy might be easy and looks better but necrobot is about 3 times faster with the same speed and almost never get softban if you use pokebuddy i get softbanned every 10 mins

**Comment by N1ghtmaree** [comment]
Y sieren use ether vpn client manager
its free

**Comment by tkkcc667788** [comment]
i agree with N1, i am using cyber ghost VPN

**Comment by N1ghtmaree** [comment]
cyberghost sucks i can find your real ip in 1 minute if i know your vpn ;p

**Comment by tkkcc667788** [comment]
wow, haha then i try to use yours

**Comment by N1ghtmaree** [comment]
but i dont think the admins of pokemon are trying to catch real ips from vpns so it might work good for pokemon

**Comment by Duero421** [comment]
The only thing i like in necrobot is that i can see the IV :+1:

**Comment by UncleAss** [comment]
i like for IV and for telegram bot.  but on 0.86 doesn t work -.-

**Comment by sierensthibault** [comment]
can someone show me their vpn client manager im a computer noob

**Comment by N1ghtmaree** [comment]
k

**Comment by N1ghtmaree** [comment]
![image](https://cloud.githubusercontent.com/assets/21019669/17649086/863620f8-622b-11e6-8458-1bfa4706f803.png)
Click VPN Gate Public VPN relay servers than click one server and wait till its loaded and than your vpn is on

**Comment by viva2020** [comment]
pokrbuddy is up...

**Comment by N1ghtmaree** [comment]
i know

**Comment by SEXYKAMO** [comment]
i can login to pokebuddy but it does not work...

**Comment by viva2020** [comment]
it work for me...

**Comment by SEXYKAMO** [comment]
[P - 13:36:51] 	 Added: (!) New settings window. Now, everything is configurable.
[P - 13:36:51] 	 Fixed: Bot now uses all available pokeballs again.
[P - 13:36:51] 	 Fixed: Start/Stop button responsability.
[P - 13:36:51] 	 Fixed: The UI now resets after stopping the bot.
[P - 13:36:51] 	 New feature: Added Minimize Button.
[P - 13:36:51] 	 New feature: Log color depending on event.
[P - 13:36:51] 	 New feature: The bot now uses Incubators
[P - 13:36:51] This bot is free to use. If you paid for it, you got scammed.
[P - 13:36:51] Feel free to donate to (PayPal): support@epicdevs.de (make sure to email me afterwards)
[P - 13:36:51] --------------------------------
[P - 13:36:53] Attempting to Login...
[P - 13:36:53] Logging in with method: PTC as Auth Provider

Why was that?

**Comment by viva2020** [comment]
why was what?

**Comment by SEXYKAMO** [comment]
cant catch any pokemons yet

**Comment by viva2020** [comment]
i got this small problem always..pokestop wont give exp everytime i started the bot but i just start it while uncheck the "Catch Pokemon" and wait till pokestop give exp..usually few minute not more than 5min.. if more than 5min u should reset your modem..

**Comment by SEXYKAMO** [comment]
that is useful. Thank you.

**Comment by viva2020** [comment]
np

**Comment by Duero421** [comment]
@SEXYKAMO  did it work?

**Comment by SEXYKAMO** [comment]
yep it works great as previous

**Comment by sierensthibault** [comment]
it keeps saying updating pokestops

**Comment by khairulpetrcech** [comment]
can i use 2 necrobot (with different google account) at the same time ?

**Comment by grimgaze** [comment]
Guys pokebuddy is working again

**Comment by N1ghtmaree** [comment]
yes you can

**Comment by grimgaze** [comment]
I can now access the web and created my account, now working perfectly

**Comment by N1ghtmaree** [comment]
You dont even have to create an account btw you can just write in all you want xd

**Comment by grimgaze** [comment]
Ah well, back to farming

**Comment by grimgaze** [comment]
Sht really nightmare? .-

**Comment by grimgaze** [comment]
Jesus.... I didnt know that. Its bcoz i still have the hangover "cant open web"

**Comment by grimgaze** [comment]
I cant run the necro so i gave up. Good thing pokebuddy is back and running again xD

**Comment by SEXYKAMO** [comment]
btw what is necro?

**Comment by N1ghtmaree** [comment]
i just walked about an hour outside playing pokemongo and didnt even got one pokemon and only 200 xp xd

**Comment by N1ghtmaree** [comment]
fuck playing legit lets bot

**Comment by N1ghtmaree** [comment]
Omg pokebuddy is so slow necrobot is about 3 times faster with the same settings lol

**Comment by grimgaze** [comment]
And it froze again

**Comment by grimgaze** [comment]
Last night was ok , does not freeze itself, now its happening again , pffffff

**Comment by grimgaze** [comment]
Lost gos again, no poke stops and all

**Comment by khairulpetrcech** [comment]
to use 2 account at the same time,do i need to use vpn ?did i can get IP ban like pokebuddy did ?

**Comment by N1ghtmaree** [comment]
yes use vpn always when botting to avoid bans

**Comment by grimgaze** [comment]
Nightmare, what is the best vpn? Can i ask yours.;)

**Comment by Rayquazaaa** [comment]
Cool!!

**Comment by N1ghtmaree** [comment]
I have Softether VPN client manager

**Comment by N1ghtmaree** [comment]
@Rab89 I need your help my Necrobot gets softbanned instantly if i just unbanned it

**Comment by UncleAss** [comment]
vpn makes you ban.  niantic see lot of traffic coming from the same ip and ban all

**Comment by grimgaze** [comment]
Nightmare in what country u connected? Cant seem to connect to any of it -.-

**Comment by UncleAss** [comment]
does someone know how works the Niantic method to detect bot users?

**Comment by Rab89** [comment]
UncleAss ive noticed a significant pattern that majority of the users being banned all use VPN software, almost as if its easily detectable, ive never used a VPN and im completely fine no ip bans or nothing

**Comment by 273849906** [comment]
anothertime

**Comment by BlitzForever** [comment]
yea mines still cant log in, has pokebuddy been banned by niantic??
